pkg/server/commands/list_objects_test.go

```diff
@@ -201,7 +214,8 @@ func TestListObjectsDispatchCount(t *testing.T) {
 
 			require.NoError(t, err)
 
-			require.Equal(t, test.expectedDispatchCount, *resp.ResolutionMetadata.DispatchCount)
+			require.Equal(t, test.expectedDispatchCount, resp.ResolutionMetadata.DispatchCounter.Load())
+			require.Equal(t, test.expectedThrottlingValue > 0, resp.ResolutionMetadata.WasThrottled.Load())
```

Comment:
Consider using [GreaterOrEqual](https://github.com/stretchr/testify/blob/v1.9.0/require/require.go#L615)

Reply:
Is this for 

`require.Equal(t, test.expectedThrottlingValue > 0, resp.ResolutionMetadata.WasThrottled.Load())`

I think we are asserting whether WasThrottled boolean was set to true depending on the provided throttle value. Not sure where I can use this method

Reply:
Ah... i see what you are trying to do - compare whether it is both expectedThrottling and WasThrottled.